x-pack/plugins/security_solution/public/alerts/components/value_lists_management_modal/table.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiBasicTable, EuiBasicTableProps, EuiText, EuiPanel } from '@elastic/eui';
+
+import { ListSchema } from '../../../../../lists/common/schemas/response';
+import { FormattedDate } from '../../../common/components/formatted_date';
+import * as i18n from './translations';
+
+type TableProps = EuiBasicTableProps<ListSchema>;
+type ActionCallback = (item: ListSchema) => void;
+
+export interface ValueListsTableProps {
+  lists: TableProps['items'];
+  loading: boolean;
+  onChange: TableProps['onChange'];
+  onExport: ActionCallback;
+  onDelete: ActionCallback;
+  pagination: Exclude<TableProps['pagination'], undefined>;
+}
+
+const buildColumns = (
+  onExport: ActionCallback,
+  onDelete: ActionCallback
+): TableProps['columns'] => [
+  {
+    field: 'name',
+    name: i18n.COLUMN_FILE_NAME,
+    truncateText: true,
+  },
+  {
+    field: 'created_at',
+    name: i18n.COLUMN_UPLOAD_DATE,
+    /* eslint-disable-next-line react/display-name */
```

Comment:
Can you add a `displayName: createdAt` to avoid disabling the linter here or no?

Ref:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md

Reply:
The complaint is because this inline function returning markup is itself technically a component. I think a naive `render: (value) => <NamedComponent value={value} />` would suffer the same issue, so you'd need to write a component that takes the full set of arguments to allow you to do `render: NamedComponent`.

I'll give it a shot.